plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts

```diff
@@ -169,12 +170,16 @@ export default function transformProps(chartProps: ChartProps): EchartsProps {
     },
     tooltip: {
       ...defaultTooltip,
-      trigger: 'axis',
-      formatter: params => {
+      trigger: richTooltip ? 'axis' : 'item',
+      formatter: (params: any) => {
```

Comment:
Can we avoid using any? Is there any better type that we can use?

Reply:
ECharts typing is out of sync with the official version that ships with `5.0`. We need to do a bigger refactoring of all echart typing references soon.